<launch>
    <param name="robot_description" command="$(find xacro)/xacro $(find mycar_description)/urdf/car.urdf.xacro" />
    <node pkg="joint_state_publisher" name="joint_state_publisher" type="joint_state_publisher" />
    <node pkg="robot_state_publisher" name="robot_state_publisher" type="robot_state_publisher" />
	<node pkg="mycar_description" name="pub_odom" type="pub_odom" output="screen"/>
    <node pkg="laser_scan_matcher" type="laser_scan_matcher_node" 
        name="laser_scan_matcher_node" output="screen">
    	<param name="use_imu" value="false"/>
    	<param name="use_odom" value="false"/>
    	<param name="use_cloud_input" value="false"/>
    	<param name="fixed_frame" value = "map"/>
		<param name="base_frame" value="base_footprint" />
    	<param name="max_iterations" value="10"/>
		<param name="publish_tf" value="false"/>
  	</node>
	

</launch>
